pkg/controllers/provisioning/scheduling/topology.go

```diff
@@ -252,6 +256,9 @@ func (t *Topology) countDomains(ctx context.Context, tg *TopologyGroup) error {
 		}
 		node := &v1.Node{}
 		if err := t.kubeClient.Get(ctx, types.NamespacedName{Name: p.Spec.NodeName}, node); err != nil {
+			if errors.IsNotFound(err) {
```

Comment:
nit: A comment on why it's safe to ignore pods for topologySpreadConstraints that have a node binding that doesn't exist?

Reply:
Done